assets/src/block-templates/issues/template.js

```diff
@@ -0,0 +1,115 @@
+import mainThemeUrl from '../main-theme-url';
+
+const item = ['core/group', {
+  className: 'd-flex',
+  backgroundColor: 'white',
+  style: {
+    border: { radius:'4px' },
+    spacing: {
+      padding: {
+        top:'32px',
+        right:'16px',
+        bottom:'32px',
+        left:'16px',
+      }
+    },
+  },
+  layout: {
+    type: 'flex',
+    flexWrap: 'nowrap',
+    justifyContent: "left",
+    orientation: 'horizontal',
+  },
+}, [
+  ['core/image', {
+    className: 'force-no-lightbox force-no-caption my-0 w-40',
```

Comment:
I've needed to add the `.w-40`, which is not necessary, but it force the width to `40px` whenever any of the images are changed. In some cases, those images are not `40x40` pixels.

Reply:
I didn't know about these classes 🤯 but it's a really good solution for the image sizes! The only "problem" is that the image's height and width in the sidebar don't match with it, but I'm not sure we can fix that 😕

Reply:
Or maybe we can using a similar technique to [what we used](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/components/Image/ImageBlockEdit.js#L27-L32) for the `is-style-rounded-` classes 🤔